modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/oauth/ocs/PlatformManagedOAuthConnectionProvider.java

```diff
@@ -216,6 +232,10 @@ private void initialiseDelegate() throws MuleException {
     }, MuleException.class, e -> e);
   }
 
+  private Optional<Class> getDelegateProviderType(ConnectionProviderModel delegateConnectionProviderModel) {
```

Comment:
pretty sure we already have an utils for this. If we don't, add it to `org.mule.runtime.module.extension.internal.util.MuleExtensionUtils`